widget/embedded/src/components/Slippage/Slippage.tsx

```diff
@@ -0,0 +1,122 @@
+import { i18n } from '@lingui/core';
+import {
+  Chip,
+  Divider,
+  InfoIcon,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@rango-dev/ui';
+import React from 'react';
+
+import {
+  MAX_SLIPPAGE,
+  MIN_SLIPPGAE,
+  SLIPPAGES,
+} from '../../constants/swapSettings';
+import { useSettingsStore } from '../../store/settings';
+
+import {
+  BaseContainer,
+  Head,
+  SlippageChipsContainer,
+  SlippageTooltipContainer,
+} from './Slippage.styles';
+
+interface TitleContainerProps {
+  title: string;
+}
+
+function TitleContainer(props: TitleContainerProps) {
```

Comment:
Why do we need a component for a simple Typography?

Reply:
It was used several times, after the file was separated, it was missed.
Done